cmd/auth/main.go

```diff
@@ -142,6 +147,13 @@ func loadConfig() config {
 		SSLRootCert: mainflux.Env(envDBSSLRootCert, defDBSSLRootCert),
 	}
 
+	lduration := mainflux.Env(envLoginDuration, defLoginDuration)
+	duration, err := strconv.Atoi(lduration)
+	if err != nil {
+		log.Fatal(err)
+	}
+	loginDuration := time.Duration(duration) * time.Minute
```

Comment:
Use 
```suggestion
loginDuration, err := time.PareDuration(mainflux.Env(envLoginDuration, defLoginDuration))
```
This also implies that duration is passed using the format with units (`1h`, `15m`, `1500s`...) which means that `defLoginDuration` needs to be changed, and it needs to be updated in README.

Reply:
Initially, the duration is passed in as a string e.g `"5"` which means 5 minutes. Then we convert this to Integer and finally time.duration

Reply:
Please use `time.PareDuration` instead and document that the duration unit needs to be passed with the value.

Reply:
Done